services/blockchain-app-registry/shared/utils/downloadRepository.js

```diff
@@ -93,6 +93,8 @@ const getRepoDownloadURL = async () => {
 };
 
 const getFileDownloadURL = async (file) => {
+	if (!file) return {};
+
```

Comment:
This might cause downstream issues. Please double check. If unnecessary, please revert.
```suggestion
```